internal/keylight/temperature.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Terin Stock
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keylight
+
+import "math"
+
+// UnmarshalTemp unmarshals Elgato API temperatures to Kelvin
+func UnmarshalTemp(elgato int) int {
```

Comment:
Unexport both, and these can live in device.go at the bottom of the file. And let's go with Convert as mentioned in chat.

Reply:
the devices tests is currently blackbox